Makefile

```diff
@@ -58,6 +58,12 @@ lint-go: tool-install
 	export pkgs="`go list -f '{{.Dir}}' ./... | grep -v /proto/`" && echo "$$pkgs" | xargs go vet -vettool=$(TOOL_BIN)/combined
 	GOGC=50 $(TOOL_BIN)/golangci-lint run -v --fix --config=./etc/.golangci.yaml
 
+lint-32bit: tool-install
+	$(TOOL_BIN)/golangci-lint run -v --build-tags no_tflite,no_pigpio --tests=false --disable-all --enable staticcheck --timeout 30m
+	go mod vendor
+	# parallelism is 1 for now because golangci-lint detects parallel runs and complains
+	etc/vendorlint.py all --linter $(abspath $(TOOL_BIN)/golangci-lint) --parallel 1
```

Comment:
`git grep vendorlint.json` currently points to the python file. you're saying it saves a step to point them to the makefile? can make this change if you think it's a maintainability issue

Reply:
It saves people a step and one can more safely change the default in the python program without affecting what git tries tracking. Admittedly that's more of a hypothetical concern, but a hygienic practice.

Though now that I'm looking more closely at this linting usage -- what's the output of vendorlint.py? I thought it was just the `vendorlint.json` file (and nothing interesting on stdout). But I don't see the github action doing anything with the `vendorlint.json` file. Should it be?

Is the default invocation of this outputting lint errors to both stdout and a json file?

Reply:
vendorlint has two steps:
1. `walk`: walk the vendors from args.root and produce a json file
2. `analyze`: process the json file + throw `FailingModules` if there are any atomicalign errors (SA1027, configurable with the INCLUDE_ISSUES global)

if invoked with `all`, as we do in make, it will do both steps

ideally we would use the golanglint-ci output instead of reprocessing, but there are cgo-related typecheck errors from golanglint-ci that afaik cannot be suppressed by configs

Reply:
--out added in 98851a40

Reply:
Ah interesting, so the file is just a temporary place to store data in. Definitely feel less strong about my former opinion on explicit/implicit. This looks good to me for what we're trying to accomplish.